rollup.config.js

```diff
@@ -5,7 +5,6 @@
 import svelte from "rollup-plugin-svelte";
 import resolve from "@rollup/plugin-node-resolve";
 import commonjs from "@rollup/plugin-commonjs";
-import livereload from "rollup-plugin-livereload";
```

Comment:
Would you kindly check if this is being used elsewhere and, if not, would you kindly remove it from the dependencies using [this command](https://docs.npmjs.com/uninstalling-packages-and-dependencies#removing-a-local-package-from-the-packagejson-dependencies)?

Reply:
I checked storybook and our normal tests, all seems well. Coming right up!